### PR TITLE
add rbac config

### PR DESCRIPTION
**Pull request by cmssczy**
fix https://github.com/kubernetes/node-problem-detector/issues/562

### CODE REVIEW

File: deployment/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: node-problem-detector
```
Comment (Random-Liu) [inline deployment/rbac.yaml:4]: NPD has a built-in role in Kubernetes `system:node-problem-detector`. We can probably use that.

**Review by Random-Liu — COMMENTED** [review]
LGTM with one nit.

File: deployment/rbac.yaml (hunk 1 above)
Comment (cmssczy) [reply deployment/rbac.yaml]: done

Comment (cmssczy) [reply deployment/rbac.yaml]: @Random-Liu PTAL

**Comment by cmssczy** [comment]
friendly ping @Random-Liu for approval

**Comment by yankay** [comment]
I had the same problem，It's nice to see that.

**Pull request closed without merging**